nb/o.n.upgrader/src/org/netbeans/upgrade/AutoUpgrade.java

```diff
@@ -91,21 +85,11 @@ private static void upgradeBuildProperties(final File sourceFolder, final String
                 userdir, regexForSelection);
     }
 
-    // the order of VERSION_TO_CHECK here defines the precedence of imports
-    // the first one will be choosen for import
-    private static final List<String> VERSION_TO_CHECK = 
-            Arrays.asList ( ".netbeans/7.1.2",  ".netbeans/7.1.1", ".netbeans/7.1", ".netbeans/7.0", ".netbeans/6.9" );//NOI18N
-    
-    // userdir on OS specific root of userdir (see issue 196075)
-    static final List<String> PRE_APACHE_NEWER_VERSION_TO_CHECK =
-            Arrays.asList ("8.2", "8.1", "8.0.2", "8.0.1", "8.0", "7.4", "7.3.1", "7.3", "7.2.1", "7.2"); //NOI18N
-     // XXX: copy to autoupgrade.pluginimporter
-    
     static final Comparator<String> APACHE_VERSION_COMPARATOR = (v1, v2) -> new SpecificationVersion(v1).compareTo(new SpecificationVersion(v2));
     
     static final List<String> APACHE_VERSION_TO_CHECK = Arrays.asList(NbBundle.getMessage(AutoUpgrade.class, "apachenetbeanspreviousversion").split(",")).stream().sorted(APACHE_VERSION_COMPARATOR.reversed()).collect(Collectors.toList());
     
-    static final List<String> NEWER_VERSION_TO_CHECK = Stream.concat(APACHE_VERSION_TO_CHECK.stream(), PRE_APACHE_NEWER_VERSION_TO_CHECK.stream()).collect(Collectors.toList());
+    static final List<String> NEWER_VERSION_TO_CHECK = APACHE_VERSION_TO_CHECK.stream().collect(Collectors.toList());
```

Comment:
`APACHE_VERSION_TO_CHECK` can be used now instead, `NEWER_VERSION_TO_CHECK` is just a copy at this point

Reply:
Doh! :laughing:  Not quite sure how that passed me by at the time ...